source/includes/extracts-atlas-search-facts.yaml

```
@@ -0,0 +1,24 @@
+ref: fact-text-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
+  Atlas Search </atlas-search/>`, for data hosted on :atlas:`MongoDB 
+  Atlas </>`. A legacy text search capability is available for users 
+  self-managing MongoDB deployments. 
+
+---
+ref: fact-regex-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
+  Atlas Search </atlas-search/>`, for data hosted on :atlas:`MongoDB 
+  Atlas </>`. If you frequently run case-insensitive regex queries 
+  (utilizing the ``i`` option), we recommend Atlas Search queries that 
+  use the :pipeline:`$search` aggregation pipeline stage. :atlas:`Atlas 
+  Search queries </reference/atlas-search/searching/>` significiantly 
```

I don't love starting the page with a dense block of copy. Maybe we should break this up a bit.

```suggestion
  use the :pipeline:`$search` aggregation pipeline stage.
  
  :atlas:`Atlas Search queries </reference/atlas-search/searching/>` significiantly 
```